internal/legacy.faustjs.org/docs/next/getting-started.mdx

```diff
@@ -72,12 +72,12 @@ Once the necessary plugins have been installed, open the `.env.local` file you c
 NEXT_PUBLIC_WORDPRESS_URL=https://headlessfw.wpengine.com
 
 # Plugin secret found in WordPress Settings->Headless
-FAUSTWP_SECRET_KEY=YOUR_PLUGIN_SECRET
```

Comment:
For backwards compatibility I think you should leave the old docs intact as the old Faust does not check the FAUST_SECRET_KEY. If the user has old version of Faust and follow this advice their builds will fail.